ArgusProbe_Beamlogic.py

```diff
@@ -190,23 +192,37 @@ def _parseBeamlogicHeader(self,header):
 
         return returnVal
 
-    def _formatZep(self,channel,timestamp,length):
+    def _formatZep(self, channel, timestamp, length, rssi):
         return [
-            0x45,0x58,
-            0x02,
-            0x01,
-            channel,
-            0x00,0x01,
-            0x01,
-            0xff,
-        ]+ \
-        [ord(b) for b in struct.pack('>Q',timestamp)]+ \
+            0x45, 0x58,     # Preamble
+            0x02,           # Version
+            0x01,           # Type (Data)
+            channel,        # Channel ID
+            0x00, 0x01,     # Device ID
+            0x01,           # CRC/LQI Mode
+            0xff,           # LQI Val
+        ] + \
+        [   # NTP Timestamp
+            ord(b) for b in struct.pack('>Q', self._get_ntp_timestamp())
+        ] + \
+        [   # Sequence number
+            0x02,0x02,0x02,0x02] + \
+        [   # Reserved Beam logic Timestamp (1/3 of us)
+            ord(b) for b in struct.pack('>Q', timestamp)] + \
+        [   # Reserved
+            rssi,
+            0x00
+        ] + \
         [
-            0x02,0x02,0x02,0x02,
-            0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,
             length,
         ]
 
+    @staticmethod
+    def _get_ntp_timestamp():
+        diff = datetime.datetime.utcnow() - datetime.datetime(1900, 1, 1, 0, 0, 0)
```

Comment:
That is Unix Timestamp. The ZEP packet expect an NTP Timestamp, that starts in 1900. It actually took me a while to realize that. https://en.wikipedia.org/wiki/Network_Time_Protocol#Timestamps